.github/workflows/pull_request.yml

```diff
@@ -26,5 +26,9 @@ jobs:
       - name: Pytest
         run: pytest python
 
+      - name: Mypy action
+        run: |
+          mypy  $(git ls-files '*.py')
+
```

Comment:
Please move this to a separate PR. Also, we need to add mypy to both the GitHub actions _and_ the pre-commit hook.

Reply:
OK.